src/passes/Inlining.cpp

```diff
@@ -48,13 +48,15 @@ namespace wasm {
 struct FunctionInfo {
   std::atomic<Index> refs;
   Index size;
-  std::atomic<bool> lightweight;
+  bool hasCalls;
+  bool hasLoops;
```

Comment:
It seems this doesn't require atomics. But we will be great if someone more experienced approve or decline this)

Reply:
Great. Thanks!